src/arm-codegen.c

```diff
@@ -85,6 +85,17 @@ void update_elf_offset(ph2_ir_t *ph2_ir)
     case OP_bit_not:
         elf_offset += 4;
         return;
+    case OP_call:
+        func = find_func(ph2_ir->func_name);
+        if (func->bbs)
+            elf_offset += 4;
+        else if (dynlink)
+            elf_offset += 12;
```

Comment:
Mention accurate PLT call offset calculation (12 bytes = 3 instructions) in comments.